packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -22,9 +22,28 @@ export async function processOutboundDocumentRetrievalResps({
   results,
 }: OutboundDocRetrievalRespParam): Promise<void> {
   try {
+    let newDocRefCount = 0;
     let issuesWithGateway = 0;
     let successDocsCount = 0;
 
+    for (const docRetrievalResp of results) {
+      newDocRefCount += docRetrievalResp.documentReference?.length ?? 0;
+    }
```

Comment:
This is done before the promises so we dont need to wait for all them to update the count